crates/wasmtime/src/store.rs

```diff
@@ -99,6 +101,7 @@ pub(crate) struct StoreInner {
     current_poll_cx: Cell<*mut Context<'static>>,
     out_of_gas_behavior: Cell<OutOfGas>,
     context_values: RefCell<HashMap<TypeId, Box<dyn Any>>>,
+    limiter: Option<Rc<dyn wasmtime_runtime::ResourceLimiter>>,
```

Comment:
I suppose one other possible option is to store `*mut dyn ResourceLimiter` in the `*mut VMContext` so we can use `Box` here instead of `Rc`, but let's only do that if the need actually arises.

Reply:
👍 I'm going to leave this as-is and we can revisit in the the future if needed.